test/file_io_test.py

```diff
@@ -0,0 +1,343 @@
+# Copyright Modal Labs 2024
```

Comment:
I think these tests are pretty good. But I think the real way to catch bugs in this is to write an integration test that fuzzes this API. It should do random writes and reads and check if those results on a regular FS are different from those results in a Sandbox.

Reply:
I'll put a server-side integration test once these changes are merged